src/SelectField/SelectField.jsx

```diff
@@ -161,70 +168,172 @@ const SelectField = React.createClass({
     };
   },
 
+  componentDidMount() {
+    if (this.props.open) {
+      /* eslint-disable */
```

Comment:
I have a couple of other things related to the `state` I want to discuss, but just as a quickie: will it not work properly in `componentWillMount`?